uaclient/config.py

```diff
@@ -405,6 +405,43 @@ def status(self, show_beta=False) -> "Dict[str, Any]":
 
         return response
 
+    def help(self, name):
+        """Return help information from an uaclient service as a dict."""
+        from uaclient.contract import get_available_resources
+        from uaclient.entitlements import ENTITLEMENT_CLASS_BY_NAME
+
+        resources = get_available_resources(self)
+        help_resource = None
+        response_dict = OrderedDict()
+        response_dict["name"] = name
+
+        for resource in resources:
+            if resource["name"] == name:
+                help_resource = resource
+                help_ent_cls = ENTITLEMENT_CLASS_BY_NAME.get(name)
+                break
+
+        if help_resource is None:
+            response_dict["error"] = "could not find service: {}".format(name)
```

Comment:
If there's no help content let's just output the rest of the YAML but have the help content be a default 

    No help available for service "xxx"